apps/ecommerce/frontend/src/components/App/Field/MultipleResources/MultipleResources.tsx

```diff
@@ -66,23 +66,26 @@ const MultipleResources = () => {
     return Array.isArray(resources) ? resources : [];
   };
 
-  const handleRemove = (index: number) => {
+  const handleRemove = (index?: number) => {
+    if (!index) return;
```

Comment:
The reason I had to make the "index" optional here is because this `ProductCard` may or may not exist within a larger context of a list at some point. And the `cardIndex` that is provided as a prop is optional, given that there is no need to provide a specific cardIndex outside of the context of the `SortableList` component. Admittedly it felt weird doing this, and I wen't back and forth a few times. Because I agree with you that the actions that handle this should require an index, but the actions themselves are so coupled to the `SortableList` component which is then using the reusable `ProductCard` so it felt justified doing this. Though I can add a default index somehow, either when from where the event is called or within these functions. What do you think?